### ISSUE TITLE
我尝试用阿里云香港和美国硅谷都不行

### ISSUE THREAD

**Issue by fangyuan99** [Ice-Hazymoon/openai-scf-proxy#61]
点击测试函数的时候都报错，放在项目里也不行，使用node16和17版本都测试了

## 测试函数报错
`发送请求失败，请稍后再试。如果多次失败，请检查您的 HTTP 函数是否禁止了跨域请求。点击[这里](https://help.aliyun.com/document_detail/71229.html#section-s91-v5a-lqw)查看 CORS 请求处理相关文档。`

## 项目报错
```
404 Not Found {
  "error": {
    "message": "Invalid URL (POST /chat/completions)",
    "type": "invalid_request_error",
    "param": null,
    "code": null
  }
}
```

**Comment by yixian3500** [comment]
和你遇到问题一样。

**Comment by suguangnet** [comment]
我的也是阿里云函数 用不了，404或者502 打开网页时会跳转到新的空白页。

**Comment by Blueness-Song** [comment]
我也是。。

**Comment by Blueness-Song** [comment]
我问了阿里云的工程师，他说这是国际网络链路问题，https方式不行，可以本地发送http请求试下，我试了换成http就好使了！

**Comment by dubaishun** [comment]
对 太牛了发自我的手机-------- 原始邮件 --------发件人： Blueness-Song ***@***.***>日期： 2023年6月26日周一 15:05收件人： Ice-Hazymoon/openai-scf-proxy ***@***.***>抄送： dubaishun ***@***.***>, Comment ***@***.***>主    题： Re: [Ice-Hazymoon/openai-scf-proxy] 我尝试用阿里云香港和美国硅谷都不行 (Issue #61)
我问了阿里云的工程师，他说这是国际网络链路问题，https方式不行，可以本地发送http请求试下，我试了换成http就好使了！

**Comment by suguangnet** [comment]
不是这个问题，打开的时候是弹出新页，和https/http没有关系。

**Comment by suguangnet** [comment]
经测试阿里云函数硅谷 弗吉尼亚 香港 链路不通，无法跳转到api.openai.com

**Comment by xcwen6** [comment]
> 经测试阿里云函数硅谷 弗吉尼亚 香港 链路不通，无法跳转到api.openai.com

阿里云新加坡可以使用,我现在就是使用新加坡的
